_redirects

```diff
@@ -41,7 +41,6 @@
 /docs/api														/docs/concepts/overview/kubernetes-api		301
 
 /docs/concepts/abstractions/controllers/garbage-collection		/docs/concepts/workloads/controllers/garbage-collection		301
-/docs/concepts/abstractions/controllers/petsets					/docs/concepts/workloads/controllers/petset		301
```

Comment:
Based on [netlify website](https://www.netlify.com/docs/redirects/), i think we should remove petset items from the redirects.

Reply:
+1

Reply:
I don't think this is correct. We don't want to remove any URLs from the left column of _redirects. We want to keep all the existing URLs in the left column, but change the target URL in the right column.

Also, for each topic that is being removed from the doc set, add the URL of the removed topic to the left column of _redirects, and put an appropriate target URL in the right column.